config/management/operational/src/command.rs

```diff
@@ -90,9 +100,11 @@ impl std::fmt::Display for CommandName {
 impl Command {
     pub fn execute(self) -> String {
         match self {
-            Command::InsertWaypoint(cmd) => {
-                format!("{:?}", cmd.execute().map(|()| "success").unwrap())
-            }
+            Command::AddValidator(cmd) => format!("{:?}", cmd.execute().unwrap()),
+            Command::InsertWaypoint(cmd) => cmd.execute().map(|()| "success".into()).unwrap(),
+            Command::ExtractPrivateKey(cmd) => cmd.execute().map(|()| "success".into()).unwrap(),
+            Command::ExtractPublicKey(cmd) => cmd.execute().map(|()| "success".into()).unwrap(),
```

Comment:
Nit: Maybe make a small function to handle the success conversion, so we don't have to copy "success" around.

Reply:
Hmm... I agree there's something awkward here, but we use these so we can actually get values for tests that leverage these operations too :/